doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -1,13 +1,24 @@
 [id='test-designer-run-test-proc']
-= Running a test scenario
+= Running the test scenarios
 
 After creating a test scenario template and defining the test scenarios, you can run the tests to validate your business rules and data.
 
 .Procedure
-. At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
-. The *Test Report* panel pops out from the right of the designer and displays the test result overview and the scenario status.
-. After the tests are executed, click *View Alerts* to open the *Alerts* panel at the bottom of the designer for the error messages.
-. In case the tests fails, review the error messages. Messages with a red cross indicate test failures where as those with a green check mark indicate successful tests.
+. To run the defined test scenarios, choose one of the following methods:
+* In the upper-right corner of your project page, click *Test* button to execute all the test scenarios available in your project inside multiple assets.
++
+.Run all the test scenarios from the project view
+image:AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
++
+* At the top of the test scenarios designer, click *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon to execute all the test scenarios defined inside a single `.scesim` file.
+* To run a single test scenario defined inside a single `.scesim` file, right click on the row that you want to execute and select *Run scenario*.
+. The *Test Report* panel pops out from the right of the designer or the project view and displays the test result overview and the scenario status.
```

Comment:
This should be a note or additional information rather than a step:

"The *Test Report* panel pops out from the right of the designer or the project view and displays the test result overview and the scenario status."

Reply:
Already discussed this issue with QE. Keeping it as a step.